frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -510,6 +511,7 @@ export class Arena {
               lastUpdated: this.lastUpdated,
               digitsAfterDecimalPoint: digitsAfterDecimalPoint,
               showPenalty: this.showPenalty,
+              showInvitedUsersFilter: self.options.contestAlias !== null,
```

Comment:
```suggestion
              showInvitedUsersFilter: options.contestAlias !== null,
```

según yo con esto no se necesita L502.

Reply:
Listo.